server/utils/response.js

```diff
@@ -49,3 +47,25 @@ export const setContentTypeFile = (res, fileName, fileSize = null, contentType =
     res.set('Content-Type', contentType)
   }
 }
+
+export const sendFile = (res, name, content, size) => {
+  setContentTypeFile(res, name, size)
+  res.write(content, 'binary')
+  res.end(null, 'binary')
+}
+
+export const sendZipFile = (res, dir, name) => {
+  if (FileUtils.existsDir(dir)) {
+    setContentTypeFile(res, name, null, contentTypes.zip)
+
+    const zip = Archiver('zip')
+    zip.pipe(res)
+    zip.directory(dir, false)
+    zip.finalize()
```

Comment:
it can be just `Archiver('zip').pipe(res).directory(dir, false).finalize()`

Reply:
default(...)(...).pipe(...).directory is not a function